buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -64,6 +69,47 @@ class BuildPlugin implements Plugin<Project>  {
             println '==================================='
         }
     }
+
+    private static void configureBuildInfo(Project project) {
+        // Make sure the global build info plugin is applied to the root project first and foremost
+        // FIXHERE: Make a proper random test seed?
+        project.rootProject.ext.testSeed = "DEADBEEF"
```

Comment:
This should get moved into the `GlobalBuildInfoPlugin`. When I fix that I'll let you know to remove this.